DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/ManageCourseController.cs

```diff
@@ -43,6 +46,17 @@ public IActionResult Index(int customisationId)
             return View(model);
         }
 
+        [HttpGet]
+        [Route("LearningPathwayDefaults/New")]
+        public IActionResult EditLearningPathwayDefaultsNew(int customisationId)
+        {
+            var model = new EditLearningPathwayDefaultsViewModel();
+
+            SetEditLearningPathwayDetailsTempData(model);

```

Comment:
I'm not sure why we are setting this here if we don't have ServiceFilters on the next couple of endpoints in the journey.

Reply:
Ah right, removed this line. Though I just tested the service filter on the auto refresh page and it doesn't seem to work. If I navigate to the auto refresh url without clicking through the learning pathway defaults page, I would expect it to redirect me to the Learning pathway defaults page, but it doesn't. Do you see what I'm missing?

Reply:
Do you have temp data around still?

Reply:
Whoops looks like those lines got added back in somehow before I committed. Removed them again, but still have the issue. Do you mean I should not be using any tempdata at all?

Reply:
I mean regarding the service filter not working as expected.

Reply:
Oh right, I forgot to clear the temp data in the index. It works now!